packages/bits/demo/src/static/typography/typography-example.component.html

```diff
@@ -5,7 +5,7 @@ <h4 class="mb-3">Core IT</h4>
 <nui-expander header="Use mixin">
     <ol>
         <li>
-            Import <strong>font-roboto.less</strong> directly like <mark>@import "@nova-ui/bits/sdk/less/mixins/font-roboto.less"</mark>
+            Import <strong>font-roboto.less</strong> directly like <mark>@import "@nova-ui/bits/dist/sdk/less/mixins/font-roboto.less"</mark>
```

Comment:
It doesn't look like the deployed bits package has a dist folder. It seems like this path should be able to stay as it was.

![image](https://user-images.githubusercontent.com/10159693/110836036-3193a700-8265-11eb-8be2-d9660e440445.png)

Reply:
So, we may need to look more into why consumers don't have access to the paths inside [this](https://github.com/solarwinds/nova/blob/3208133f89247f4bf4ca48c89c88e6958f7b26c8/packages/bits/src/styles/mixins/font-roboto.less#L6) file.